release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -802,9 +816,23 @@ include::snippets/olmv1-cli-only.adoc[]
 
 For more information, see xref:../operators/olm_v1/index.adoc#olmv1-about[About Operator Lifecycle Manager 1.0].
 
+[id="ocp-4-15-deprecation"]
+==== Deprecation schema for Operator catalogs
+
+The optional `olm.deprecations` schema defines deprecation information for Operator packages, bundles, and channels in a file-based catalog. Operator authors can use this schema in a `deprecations.yaml` file to provide relevant messages about their Operators, such as support status and recommended upgrade paths, to users running those Operators from a catalog. After the Operator is installed, any specified messages can be viewed as status conditions on the related Subscription object.
+
+For information on the `olm.deprecations` schema, see xref:../operators/understanding/olm-packaging-format.adoc#olm-deprecations-schema_olm-packaging-format[Operator Framework packaging format].
+
 [id="ocp-4-15-osdk"]
 === Operator development
 
+[id="ocp-4-15-osdk-cco-azure"]
+==== Token authentication for Operators on cloud providers: {azure-id}
+
+With this release, Operators managed by Operator Lifecycle Manager (OLM) can support token authentication when running on Azure clusters configured for {azure-id}. The Cloud Credential Operator (CCO) is updated to semi-automate provisioning certain short-term credentials, provided that the Operator author has enabled their Operator to support {azure-id}.
```

Comment:
just a suggestion to avoid past tense:
```suggestion
With this release, Operators managed by Operator Lifecycle Manager (OLM) can support token authentication when running on Azure clusters configured for {azure-id}. Updates to the Cloud Credential Operator (CCO) enable semi-automated provisioning of certain short-term credentials, provided that the Operator author has enabled their Operator to support {azure-id}.
```